app/services/idv/session.rb

```diff
@@ -73,6 +73,7 @@ def complete_session
 
     def complete_profile
       profile.verified_at = Time.zone.now
+      profile.vendor = vendor
```

Comment:
Did you mean to add this here? Perhaps this was an unintentional consequence of the rebase with master? This line doesn't exist in master.